src/lib/containers/UserCardSmall.tsx

```diff
@@ -110,6 +111,7 @@ export const UserCardSmall: React.FunctionComponent<UserCardSmallProps> = ({
 
       <a
         ref={target}
+        href={disableLink ? undefined : link}
```

Comment:
On click, does it both change the current window to the link url and open it up in a new window?

Reply:
(looking at the window.open(link, '_blank') in the click handler)

Reply:
Just opens the profile in a new window. `event.preventDefault()` prevents the link from opening in the new window (at least in Firefox, Chrome, and Safari)